### PR TITLE
EO-850 Improve RV batch upload visuals

### PR DESCRIPTION
**Pull request by jonambas**
Mocks: https://sparkpost.invisionapp.com/share/KESDZMGX4JC#/367407129_Recipient_Validation_Home_-_Lists

### What Changed
- Mostly just visual changes
- Bumps matchbox-icons for `FileType` icon
- Submit is handled on input change instead of form submit

### How To Test
- Log into appteam
- Test behavior: dragging, & clicking and browsing
- Can modify `maxRecipVerifUploadSizeBytes`  in `config/default` to test error behavior

### To Do
- [ ] Feedback

### CODE REVIEW

File: src/pages/recipientValidation/components/FileUploadWrapper.module.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
+@import '~@sparkpost/matchbox/src/styles/config.scss';
+@import '~@sparkpost/matchbox/src/components/TextField/TextField.module.scss';
```
Comment (jonambas) [inline src/pages/recipientValidation/components/FileUploadWrapper.module.scss]: Remove

File: src/pages/recipientValidation/RecipientValidationPage.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ import RVDisabledPage from './components/RVDisabledPage';
 import ConditionSwitch, { Case, defaultCase } from 'src/components/auth/ConditionSwitch';
 
 const tabs = [
-  { content: 'Validate A List' },
+  { content: <span style={{ padding: '0 18px' }}>List</span> },
```
Comment (rajumsys) [inline src/pages/recipientValidation/RecipientValidationPage.js]: can it be moved to css file?

File: src/pages/recipientValidation/components/tests/FileUploadWrapper.test.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+};
+
+it('renders correctly', () => {
+  const wrapper = shallow(<FileUploadWrapper {...defaults} />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('renders and error correctly', () => {
+  const wrapper = shallow(<FileUploadWrapper {...defaults} meta={{ error: 'oh no', touched: true }}/>);
+  expect(wrapper.find('Error')).toMatchSnapshot();
```
Comment (rajumsys) [inline src/pages/recipientValidation/components/tests/FileUploadWrapper.test.js]: looks like we can easily do inline assertion here? 
```
expect(wrapper.find('Error')).toExist();
expect(wrapper.find('Error').prop('error')).toEqual('oh no');
```

**Comment by jonambas** [comment]
I agree the UX is not ideal but this is just a small step for now until the API fully supports history

**Comment by jonambas** [comment]
🙌

**Pull request merged**